File: DiceProblem.py
def diceProblem(n):
    return 7 - n

n = int(input("Hãy nhập một số nguyên bất kì từ 1 đến 6: "))
if n < 1 or n > 6:
    print("\nHãy nhập số nguyên từ 1 đến 6!")
else:
    print(f"\nMặt đối diện của {n} là {diceProblem(n)}")

File: DiceProblem.c
#include <stdio.h>

int diceProblem(int n){
    return 7 - n;
}

int main(){
    int n;
    printf("Hãy nhập 1 số nguyên bất kì từ 1 đến 6: ");
    scanf("%d", &n);

    if(n <=0 || n >=7){
        printf("Hãy nhập số nguyên từ 1 đến 6!\n");
    }   else{
        int result = diceProblem(n);
        printf("Mặt đối diện của %d là %d\n", n, result);
    }
    return 0;
}
